src/components/map/layers/objects/NodeMarker.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from 'react';
```

Comment:
The folder name 'objects' is maybe a bit too generic, or what do you think? We are using object-oriented programming, so the name 'object' can basically mean anything in my opinion.